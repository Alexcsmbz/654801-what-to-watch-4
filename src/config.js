export const moviePageTabs = [
  {
    name: `Overview`,
  },
  {
    name: `Details`,
  },
  {
    name: `Reviews`,
  },
];

export const RoutePath = {
  MAIN: `/`,
  LOGIN: `/login`,
  MOVIE: `/films/:id`,
  MYLIST: `/mylist`,
  REVIEW: `/films/:id/review`,
};
