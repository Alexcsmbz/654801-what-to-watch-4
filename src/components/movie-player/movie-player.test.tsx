import MoviePlayer from './movie-player';
import {testMovie} from 'config';

describe(`MoviePlayer snapshot test`, () => {
  it(`MoviePlayer should render movie info`, () => {
    const tree = renderer.create(
        <MoviePlayer movie={testMovie} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
