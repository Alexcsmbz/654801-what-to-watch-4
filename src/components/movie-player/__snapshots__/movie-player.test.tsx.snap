// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlayer snapshot test MoviePlayer should render movie info 1`] = `
<video
  controls={true}
  height="175"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="previewImage"
  preload="on"
  width="280"
>
  <source
    src="previewVideoLink"
    type="video/mp4"
  />
</video>
`;
